ManatiCR/vitaminau
# VIT-20: Form Entrepreneurship Support

# Task [VIT-20](https://manati.atlassian.net/browse/VIT-20)
Creation of the "Apoyo al Emprendimiento" webform.

# Installation
- git pull in this branch
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user
  - Username: admin
  - Password: admin
- Go to the following web path: 
  - http://vitaminau.docker/form/webform-entrepreneurship-support
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-20](https://manati.atlassian.net/browse/VIT-20?atlOrigin=eyJpIjoiZDEzZWVhNTU4NjBjNDkwYTk5MzVjNzBlOTYwYTc1ZmEiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save button
- Once you press it your information should be saved and the form should redirect you to a thank you page
- To complete a form you have to add a term in the "unit" taxonomy, at this address: ``http://vitaminau.docker/admin/structure/taxonomy/manage/unidad/add`` you can add an element with the name "test".
- Save your draft with the button Save Draft and log in again in another browser or incognito, a "Este formulario ha sido precargado con la información almacenada en el borrador." message should appear.
- if you want check your response or your draft you can go this path ``http://vitaminau.docker/admin/structure/webform/manage/webform_entrepreneurship_support/results/submissions``  and search your answer.
- To test the required fields, leave a blank, which has a red "*" and the same form will tell you that it has not been completed yet.


# Acceptance criteria
- Only authenticated users can access the form.
- The following fields are not required
  - Contrapartida requerida
  - Trayectoria > Otro
  - Restricciones
  - ¿Algo que agregar?
  - PDF relevante
  - ¿Cómo calificaría este formulario?
  - ¿Qué mejoraría?
- When the required fields are empty, you should see a message, indicating that it must be filled
